demos/projects/ESPRESSIF/esp32/components/azure-sdk-for-c/CMakeLists.txt

```diff
@@ -0,0 +1,26 @@
+set(ROOT_PATH
+    ${CMAKE_CURRENT_LIST_DIR}/../../../../../..
+)
+
+set(AZURE_IOT_MIDDLEWARE_FREERTOS
+    ${ROOT_PATH}/libs/azure-iot-middleware-freertos
+)
+
+set(AZURE_SDK_FOR_C_PATH 
+    ${AZURE_IOT_MIDDLEWARE_FREERTOS}/libraries/azure-sdk-for-c/sdk
+)
+
+file(GLOB_RECURSE COMPONENT_SOURCES
+    ${AZURE_SDK_FOR_C_PATH}/src/azure/*.c  
+)
+
+list(FILTER COMPONENT_SOURCES EXCLUDE REGEX ".*(curl|win32).*")
+
+set(COMPONENT_INCLUDE_DIRS
+    ${AZURE_IOT_MIDDLEWARE_FREERTOS}/.github/config
```

Comment:
embedded sdk does not have concept of config, we do not need this.

Reply:
Removed.